pubspec.yaml

```diff
@@ -10,17 +10,22 @@ dependencies:
   html: ^0.15.0
   flutter:
     sdk: flutter
-  cached_network_image: ^3.0.0
-  matrix_link_text: ^1.0.2
-  flutter_math_fork: ^0.4.1
+  cached_network_image: ^3.2.1
+  matrix_link_text: ^1.1.0
+  flutter_math_fork: ^0.6.3
   flutter_highlight: ^0.7.0
-  isolate: ^2.0.3
   highlight: ^0.7.0
-  crypto: ^3.0.1
-  csslib: ^0.17.1
+  crypto: ^3.0.2
+  csslib: ^0.17.2
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
 
-flutter:
+  flutter_lints: ^2.0.1
+
+dependency_overrides:
```

Comment:
not a fan of this, please PR to matrix_link_text instead so that soru can publish a new version

Reply:
https://github.com/Sorunome/matrix_link_text

Reply:
ah yeah, saw it now